Packs/Gmail/pack_metadata.json

```diff
@@ -8,7 +8,7 @@
     "email": "",
     "created": "2020-04-14T00:00:00Z",
     "categories": [
-        "Email Gateway"
+        "IT Services"
```

Comment:
Cloud Services?

Reply:
it is indeed only a cloud service. fixing.